packages/mason/lib/src/bundler.dart

```diff
@@ -42,22 +42,25 @@ void unpackBundle(MasonBundle bundle, Directory target) {
   if (license != null) _unbundleFile(license, target.path);
 }
 
-/// Generates a [MasonBundle] from the provided [brick] directory.
-MasonBundle createBundle(Directory brick) {
-  final brickYamlFile = File(path.join(brick.path, BrickYaml.file));
+/// Generates a [MasonBundle] from the provided [brickPath] directory.
+MasonBundle createBundle(String brickPath) {
```

Comment:
Is this change necessary? it's a breaking change and it doesn't appear to be necessary can we revert it?

Reply:
Removing it 🗡️